Source/player.h

```diff
@@ -900,6 +900,11 @@ struct Player {
 
 		return (type == leftHandItem._itype && leftHandItem._iStatFlag) || (type == rightHandItem._itype && rightHandItem._iStatFlag);
 	}
+
+	inline bool hasNoLife() const

```

Comment:
`inline` shouldn't be necessary for member functions.
```suggestion
	bool hasNoLife() const
```